contrib/database/sql/propagation_test.go

```diff
@@ -288,6 +288,33 @@ func TestDBMTraceContextTagging(t *testing.T) {
 	}
 }
 
+func TestDBMPropagation_PreventFullMode(t *testing.T) {
```

Comment:
It might make sense to have a test that goes through the implicit driver registration (i.e. when a user doesn't call `Register` and uses `Open` directly?

Reply:
nice catch! adding it 👍

Reply:
🙇